zitadel script fails to get latest version from github

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

zitadel

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/zitadel.sh)"

### 📝 Provide a clear and concise description of the issue.

```
 ⠋ Installing Zitadel 
[ERROR] in line 42: exit code 0: while executing command RELEASE=$(curl -fsSL https://github.com/zitadel/zitadel/releases/latest | grep location: | cut ') '/' -f 8 | tr -d '
```

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

run the bash script for zitadel

### ❌ Paste the full error output (if available).

running bash script on pve host

### 🖼️ Additional context (optional).

I found that this like works.
```
RELEASE=$(curl -fsSL https://github.com/zitadel/zitadel/releases/latest | grep -m1 "/releases/tag/v" | cut -d '/' -f 8 | tr -d '"')
```

--- a/ct/zitadel.sh
+++ b/ct/zitadel.sh
@@ -27,7 +27,7 @@ function update_script() {
         msg_error "No ${APP} Installation Found!"
         exit
     fi
-    RELEASE=$(curl -fsSL https://github.com/zitadel/zitadel/releases/latest | grep location: | cut -d '/' -f 8 | tr -d '\r')
+    RELEASE=$(curl -fsSL https://api.github.com/repos/zitadel/zitadel/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
     if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt | grep -oP '\d+\.\d+\.\d+')" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
         msg_info "Stopping $APP"
         systemctl stop zitadel
@@ -35,7 +35,7 @@ function update_script() {
 
         msg_info "Updating $APP to ${RELEASE}"
         cd /tmp || exit
-        curl -fsSL https://github.com/zitadel/zitadel/releases/download/"$RELEASE"/zitadel-linux-amd64.tar.gz | tar -xz
+        curl -fsSL "https://github.com/zitadel/zitadel/releases/download/v$RELEASE/zitadel-linux-amd64.tar.gz | tar -xz
         mv zitadel-linux-amd64/zitadel /usr/local/bin
         $STD zitadel setup --masterkeyFile /opt/zitadel/.masterkey --config /opt/zitadel/config.yaml --init-projections=true
         echo "${RELEASE}" >/opt/${APP}_version.txt

--- a/ct/zitadel.sh
+++ b/ct/zitadel.sh
@@ -20,39 +20,39 @@ color
 catch_errors
 
 function update_script() {
-    header_info
-    check_container_storage
-    check_container_resources
-    if [[ ! -f /etc/systemd/system/zitadel.service ]]; then
-        msg_error "No ${APP} Installation Found!"
-        exit
-    fi
-    RELEASE=$(curl -fsSL https://api.github.com/repos/zitadel/zitadel/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
-    if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt | grep -oP '\d+\.\d+\.\d+')" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
-        msg_info "Stopping $APP"
-        systemctl stop zitadel
-        msg_ok "Stopped $APP"
-
-        msg_info "Updating $APP to ${RELEASE}"
-        cd /tmp || exit
-        curl -fsSL "https://github.com/zitadel/zitadel/releases/download/v$RELEASE/zitadel-linux-amd64.tar.gz | tar -xz
-        mv zitadel-linux-amd64/zitadel /usr/local/bin
-        $STD zitadel setup --masterkeyFile /opt/zitadel/.masterkey --config /opt/zitadel/config.yaml --init-projections=true
-        echo "${RELEASE}" >/opt/${APP}_version.txt
-        msg_ok "Updated $APP to ${RELEASE}"
-
-        msg_info "Starting $APP"
-        systemctl start zitadel
-        msg_ok "Started $APP"
-
-        msg_info "Cleaning Up"
-        rm -rf /tmp/zitadel-linux-amd64
-        msg_ok "Cleanup Completed"
-        msg_ok "Update Successful"
-    else
-        msg_ok "No update required. ${APP} is already at ${RELEASE}"
-    fi
+  header_info
+  check_container_storage
+  check_container_resources
+  if [[ ! -f /etc/systemd/system/zitadel.service ]]; then
+    msg_error "No ${APP} Installation Found!"
     exit
+  fi
+  RELEASE=$(curl -fsSL https://api.github.com/repos/zitadel/zitadel/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt | grep -oP '\d+\.\d+\.\d+')" ]] || [[ ! -f /opt/${APP}_version.txt ]]; then
+    msg_info "Stopping $APP"
+    systemctl stop zitadel
+    msg_ok "Stopped $APP"
+
+    msg_info "Updating $APP to ${RELEASE}"
+    cd /tmp || exit
+    curl -fsSL "https://github.com/zitadel/zitadel/releases/download/v$RELEASE/zitadel-linux-amd64.tar.gz" | tar -xz
+    mv zitadel-linux-amd64/zitadel /usr/local/bin
+    $STD zitadel setup --masterkeyFile /opt/zitadel/.masterkey --config /opt/zitadel/config.yaml --init-projections=true
+    echo "${RELEASE}" >/opt/${APP}_version.txt
+    msg_ok "Updated $APP to ${RELEASE}"
+
+    msg_info "Starting $APP"
+    systemctl start zitadel
+    msg_ok "Started $APP"
+
+    msg_info "Cleaning Up"
+    rm -rf /tmp/zitadel-linux-amd64
+    msg_ok "Cleanup Completed"
+    msg_ok "Update Successful"
+  else
+    msg_ok "No update required. ${APP} is already at ${RELEASE}"
+  fi
+  exit
 }
 
 start

--- a/install/zitadel-install.sh
+++ b/install/zitadel-install.sh
@@ -39,8 +39,8 @@ $STD sudo -u postgres psql -c "CREATE DATABASE $DB_NAME OWNER $DB_ADMIN_USER;"
 msg_ok "Installed PostgreSQL"
 
 msg_info "Installing Zitadel"
-RELEASE=$(curl -fsSL https://github.com/zitadel/zitadel/releases/latest | grep location: | cut -d '/' -f 8 | tr -d '\r')
-curl -fsSL https://github.com/zitadel/zitadel/releases/download/$RELEASE/zitadel-linux-amd64.tar.gz | tar -xz
+RELEASE=$(curl -fsSL https://api.github.com/repos/zitadel/zitadel/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+curl -fsSL "https://github.com/zitadel/zitadel/releases/download/v$RELEASE/zitadel-linux-amd64.tar.gz" | tar -xz
 mv zitadel-linux-amd64/zitadel /usr/local/bin
 msg_ok "Installed Zitadel"
 

